www/novj35/firma.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Apple</title>
+</head>
+
+<body>
+    <header>
+        <span>Apple</span>
+        <nav>
+            <li><a href="">Domů</a></li>
```

Comment:
Tady je špatné použití tagu &lt;li&gt;, protože musí být pod &lt;ul&gt;

Tahle chyba se objeví při validaci na https://validator.w3.org

Takže prosím zkus to zvalidovat a opravit